centos_ci.sh

```diff
@@ -304,12 +312,17 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags ~coolstore -format pretty"
+  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags proxy -format pretty"
```

Comment:
@budhrg This is just to test if this passes centos-ci, on merge it should be probably changed to `MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags ~coolstore,~proxy -format pretty"`. @hferentschik